mathics/builtin/calculus.py

```diff
@@ -314,6 +392,7 @@ def __init__(self, *args, **kwargs):
         super(Derivative, self).__init__(*args, **kwargs)
 
     def to_sympy(self, expr, **kwargs):
+        print("calling to_sympy")
```

Comment:
We probably want to remove here. 

Ideally I'd like to figure out a way where we could not have our commit history fill of this is the place where we removed the debug print. Squashing commits might be one way.

Reply:
I squashed this, but it seems that this one escaped to my sight... In any case, I found that this function is never called, except in some tests...